src/pykeen/models/unimodal/rgcn/model.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of the R-GCN model."""
+
+import logging
+from os import path
+from typing import Any, Mapping, Optional
+
+import torch
+from class_resolver import Hint, Resolver
+from torch import nn
+
+from .decompositions import Decomposition, decomposition_resolver
+from .weightings import EdgeWeighting, edge_weight_resolver
+from ...nbase import ERModel, EmbeddingSpecificationHint
+from ....nn import EmbeddingSpecification, RepresentationModule
+from ....nn.modules import Interaction, interaction_resolver
+from ....triples import TriplesFactory
+from ....typing import Initializer, RelationRepresentation
+
+__all__ = [
+    "RGCN",
+]
+
+logger = logging.getLogger(name=path.basename(__file__))
+
+
+class Bias(nn.Module):
+    """A module wrapper for adding a bias."""
+
+    def __init__(self, dim: int):
+        """Initialize the module.
+
+        :param dim: >0
+            The dimension of the input.
+        """
+        super().__init__()
+        self.bias = nn.Parameter(torch.empty(dim), requires_grad=True)
+        self.reset_parameters()
+
+    def reset_parameters(self):
+        """Reset the layer"s parameters."""
+        nn.init.zeros_(self.bias)
+
+    # pylint: disable=arguments-differ
```

Comment:
is pylint comment necessary?

Reply:
done in d8dda61b3f3c8bb8e9b56ce4d246bd3f810f4abf